azurelinuxagent/common/utils/textutil.py

```diff
@@ -323,10 +323,10 @@ def b64encode(s):
     return base64.b64encode(s)
 
 
-def b64decode(s):
+def b64decode(s, encoding='utf-8'):
```

Comment:
I would be careful about changes here. Make sure you really need them

Reply:
Ditto.  There by dragons.  I am curious why you needed this change, or where you just trying to debug through it?

---
In reply to: [192194876](https://github.com/Azure/WALinuxAgent/pull/1171#discussion_r192194876) [](ancestors = 192194876)